packages/extension-ui/src/Popup/AuthManagement/AccountManagement.tsx

```diff
@@ -102,7 +99,7 @@ function AccountManagement({ className, location: { search } }: Props): React.Re
           {hierarchy.length > 0 && (
             <CustomButtonArea>
               <Button
-                onClick={_onCancel}
+                onClick={() => history.goBack()}
```

Comment:
```
onAction('..')
```

Reply:
`goBack()` is intentional here - that's because this screen can from now on be reached from more than 1 place. Actually, the fact that there was `onAction` in there in the first place was kind of bad semantically as well, I think.

Reply:
Oh crap, no I get it - disregard my comment.